src/main/java/se/sundsvall/supportmanagement/integration/db/model/ErrandEntity.java

```diff
@@ -74,6 +74,12 @@ public class ErrandEntity implements Serializable {
 	@Column(name = "status_tag")
 	private String statusTag;
 
+	@Column(name = "resolution")
+	private String resolution;
+
+	@Column(name = "description", length = 5000)
```

Comment:
Do we know that 5000 chars is sufficient?

If no: 

> Solution 1: Remove length-attribute and annotate with `@Lob`? This will translate to datatype LONGTEXT in mariaDB.
> Solution 2: Remove length-attribute and add columnDefinition-attribute `columnDefinition = "TEXT/MEDIOUMTEXT/LONGTEXT/or whatever datatype you want..."` (not as portable as `@Lob` though)
> 

If yes: 

> Reflect the length-constraint in Errand API-model.